### ISSUE TITLE
privacy switches 

### ISSUE THREAD

**Issue by shapironick** [publiclab/wherewebreathe#45]
Maybe I'm off here but I think there may be some confusion about there being two switches in Jeff's diagram in issue #2. I thought that the two privacy switches in the mock up were actually representing the two options (on/off) of a single switch. I'm think they should be merged into a single switch as I don't think the second switch should exist, because what would even happen if they opted out of letting the research team see the data and opted out of data being public?  The data would just disappear once they entered it?

I like Melissa’s urge on the site to use language other than “data” as it is a little vague and may be somewhat alien to our average user.  

So maybe we could have the two toggles be: 

a)	Your questionnaire answers are currently shared only with the WhereWeBreathe research team and will not be visible publicly. 
b)	Your questionnaire answers can be seen by the research team and anyone on the internet. (Only your chosen username will be shown along with your answers).

Option A would be the preset. 

Also, some users may not recognize the switch as something they can interact with, so maybe below the switch we can say for option a) you can slide this switch to the right to share publicly and b) you can slide this switch to the right to share exclusively with the research team. We could also have a little arrow like in jeff's diagram above. 

After clicking update privacy settings it would be nice to be automatically sent to the questionnaire. 

@mmmelissa @jywarren

**Comment by shapironick** [comment]
I don't think we have the three different stages of privacy of the forums yet in the site flow. I think this should come after the study and before the first forum. 

Text, here: http://publiclab.org/wiki/wherewebreathe#Forum+Privacy 

You are about to enter the Forums where you can tell you story in full, see live graphs of survey data and connects with other people facing similar issues as you.

You have 3 options for who can see what you post in the forums. Please take a chance to select one. You can change your selection at any time, which will change the accessibility of your posts from that moment forward.

(multiple choice)

Community members and the research team can see your posts.

Community members and not the research team* can see your posts.

The research team and not the community members can see your posts.

*The system administrator will be able to see content that has been deemed inappropriate by community facilitators (learn more about community facilitators here (link)).

[If they select an option with community members in it--this pops up: Please only share what you would feel comfortable sharing in a public but anonymous way, only the research team is bound to strict levels of confidentiality]

**Comment by shapironick** [comment]
@jywarren we still need some text about the movement between spaces of differing privacy and this would also be embedded in a visual logic. e.g. when you see this symbol/ color etc you know that you are sharing only with ....

could you take a crack at that?

**Comment by gretchengehrke** [comment]
Can I go in and make a couple of wording edits to the Forum+Privacy texts?  
I want to change to: 

You are about to enter the Forums where you can tell your story in full, see live graphs of survey data and connect with other people facing similar issues.

You have 3 options for who can see what you post in the forums. Please take this opportunity to select one. You can change your selection at any time, which will change the accessibility of your posts from that moment forward.

(multiple choice)

Community members and the research team can see your posts.

Community members and not the research team can see your posts.*

The research team and not the community members can see your posts.

*The system administrator will be able to see content that has been deemed inappropriate by community facilitators (learn more about community facilitators here (link)).

[If they select an option with community members in it--this pops up: Please only share what you would feel comfortable sharing in a public (but anonymous) way; only the research team is bound to strict levels of confidentiality]

**Comment by shapironick** [comment]
Please do!
